resources/l10n/de/LC_MESSAGES/loot.po

```diff
@@ -805,87 +813,95 @@ msgstr "Hinzufügen/Entfernen"
 msgid "Bash Tag"
 msgstr "Bash Tag"
 
-#: src/gui/qt/settings/game_tab.cpp:123
+#: src/gui/qt/settings/game_tab.cpp:57
+msgid "Browse..."
+msgstr "Durchsuchen..."
+
+#: src/gui/qt/settings/game_tab.cpp:157
 #: src/gui/qt/settings/new_game_dialog.cpp:88
 msgid "Base Game"
 msgstr "Basis-Videospiel"
 
-#: src/gui/qt/settings/game_tab.cpp:124
+#: src/gui/qt/settings/game_tab.cpp:158
 #: src/gui/qt/settings/new_game_dialog.cpp:89
 msgid "Is instance of base game"
 msgstr "Ist Instanz des Basis-Videospiels"
 
-#: src/gui/qt/settings/game_tab.cpp:125
+#: src/gui/qt/settings/game_tab.cpp:159
 #: src/gui/qt/settings/new_game_dialog.cpp:90
 msgid "LOOT Folder"
 msgstr "LOOT Ordner"
 
-#: src/gui/qt/settings/game_tab.cpp:126
+#: src/gui/qt/settings/game_tab.cpp:160
 msgid "Main Master Plugin"
 msgstr "Haupt Master Plugin"
 
-#: src/gui/qt/settings/game_tab.cpp:127
+#: src/gui/qt/settings/game_tab.cpp:161
 msgid "Minimum Header Version"
 msgstr "Minimale Kopfzeilenversion"
 
-#: src/gui/qt/settings/game_tab.cpp:128
+#: src/gui/qt/settings/game_tab.cpp:162
 msgid "Masterlist Source"
 msgstr "Masterlist Quelle"
 
-#: src/gui/qt/settings/game_tab.cpp:129
+#: src/gui/qt/settings/game_tab.cpp:163
 msgid "Install Path"
 msgstr "Installationspfad"
 
-#: src/gui/qt/settings/game_tab.cpp:130
+#: src/gui/qt/settings/game_tab.cpp:164
 msgid "Install Path Registry Keys"
 msgstr "Installations-Pfad Registry-key"
 
-#: src/gui/qt/settings/game_tab.cpp:131
+#: src/gui/qt/settings/game_tab.cpp:165
 msgid "Local AppData Path"
 msgstr "Lokaler AppData Ordnerpfad"
 
-#: src/gui/qt/settings/game_tab.cpp:133
+#: src/gui/qt/settings/game_tab.cpp:167
 msgid "Delete game"
 msgstr "Lösche Videospiel"
 
-#: src/gui/qt/settings/general_tab.cpp:145
+#: src/gui/qt/settings/general_tab.cpp:152
 msgid "Default Game"
 msgstr "Standard-Videospiel"
 
-#: src/gui/qt/settings/general_tab.cpp:147
+#: src/gui/qt/settings/general_tab.cpp:154
 msgid "Theme"
 msgstr "Design"
 
-#: src/gui/qt/settings/general_tab.cpp:148
+#: src/gui/qt/settings/general_tab.cpp:155
 msgid "Update masterlist before sorting"
 msgstr "Aktualisiere Masterlist vor dem Sortieren"
 
-#: src/gui/qt/settings/general_tab.cpp:149
+#: src/gui/qt/settings/general_tab.cpp:156
 msgid "Check for LOOT updates on startup"
 msgstr "Prüfe auf LOOT Updates beim Programmstart"
 
-#: src/gui/qt/settings/general_tab.cpp:150
+#: src/gui/qt/settings/general_tab.cpp:157
 msgid "Enable debug logging"
 msgstr "Aktivere Debug-Protokollierung"
 
-#: src/gui/qt/settings/general_tab.cpp:151
+#: src/gui/qt/settings/general_tab.cpp:158
 msgid "Masterlist prelude source"
 msgstr "Masterlist Prelude Quelle"
 
-#: src/gui/qt/settings/general_tab.cpp:154
+#: src/gui/qt/settings/general_tab.cpp:160
+msgid "Display dialog when sorting makes no changes"
+msgstr "Zeige Dialog wenn Sortieren keine Änderungen macht"
```

Comment:
@pStyl3 Does this make sense for where it is used?

Reply:
I think your translation is okay. See [here](https://github.com/loot/loot/issues/1729#issuecomment-1106704528) for more information on the subject.